### DISCUSSION TITLE
Allow for option to prevent rechecking on restart

### DISCUSSION THREAD

**Discussion by GodTamIt** [haugene/docker-transmission-openvpn#2330] [Ideas]
Unless I'm missing something big, there doesn't seem to be a way to prevent rechecking of the entire corpus on restart.

If you're tracking a lot of items, this can take quite a long time. It would be very helpful to have some way to prevent this from happening on every restart. This might be something that is on the transmission side as well :shrug:

**Comment by pkishino** [comment]
This does not normally happen.. did you recently change versions? Please see 4.0 release thread regarding diff transmission versions
